tests/test_blog.py

```
@@ -97,6 +97,38 @@ def mock_path_side_effect(path_str):
             BlogDigestBuilder()
             assert temp_blogs_dir.exists()
     
+    def test_r2publisher_integration(self):
+        """Test that R2Publisher is properly imported and can be instantiated."""
+        builder = BlogDigestBuilder()
```

Ruff (0.12.2)

Local variable `builder` is assigned to but never used

Remove assignment to unused variable `builder`

(F841)